### PR TITLE
python3Packages.{panel,bokeh}: update packages

### PR DESCRIPTION
**Pull request by FRidh**
###### Motivation for this change

With newer versions of panel js assets need to be fetched. Version 2.3 of bokeh is not compatible anymore with older panel versions so it is time to upgrade. I've tested with several bokeh/panel applications that were otherwise broken by the update of these packages. As far as I can tell, everything is good now.

Works around https://github.com/holoviz/panel/issues/1819.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jonringer** [comment]
sigh... node modules ;(

File: pkgs/development/python-modules/panel/default.nix [hunk 1]
```diff
@@ -8,19 +8,39 @@
...
   pname = "panel";
-  version = "0.9.7";
-  # Version 10 attempts to download models from the web during build-time
-  # https://github.com/holoviz/panel/issues/1819
+  version = "0.11.1";
 
+  # Don't forget to also update the node packages
+  # 1. retrieve the package.json file
+  # 2. nix shell nixpkgs#nodePackages.node2nix
+  # 3. node2nix
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/panel/default.nix:23]: Maybe an update script for this should be written.

**Comment by SuperSandro2000** [comment]
@ofborg eval

File: pkgs/development/python-modules/panel/default.nix (hunk 1 above)
Comment (FRidh) [reply pkgs/development/python-modules/panel/default.nix:23]: Yes that would be a nice thing to have.

**Pull request merged**